Message/AppleMessage.php

```diff
@@ -75,11 +75,12 @@ public function __construct($identifier = NULL)
     /**
      * Sets the message. For iOS, this is the APS alert message
      *
+     * @param $title
      * @param $message
      */
-    public function setMessage($message)
+    public function setMessage($title, $message)
```

Comment:
If this is the only place you need the title, then consider adding a new method on the `AppleMessage` class only, rather than making the changes to all the other message classes, where it is irrelevant.